examples/update-demo/kitten-rc.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ReplicationController
+id: update-demo-kitten
```

Comment:
Since you're running this through sed anyway, why not parameterize kitten vs. nautilus (e.g., IMAGE_VERSION)?

Really we should be using jinja, jq, or something, but sed will work and doesn't create additional package dependencies.